app/api/v1/models/Redflags.py

```diff
@@ -21,11 +21,6 @@ def save(self, title, description, location, incident_type):
     def get_flags(self):
         return self.db
 
-
-class SingleFlagModel():
-    def __init__(self):
-        self.db = red_flags
-
     def getSingleFlag(self, flag_id):
```

Comment:
Use python naming convention. For all your function names 
```suggestion
    def get_single_flag(self, flag_id):
```